utils/resizingConstraints.js

```diff
@@ -0,0 +1,33 @@
+const containsAllItems = (needles, haystack) => needles.every(needle => haystack.includes(needle));
```

Comment:
What do you think about putting this in the **/utils/maps.js** file? It would make that file a bit longer, but other similar integer maps live there. Or maybe we should split each map to its own file?

Reply:
I also move to `utils/maps.js` the `resizingConstraintsMap`.